src/Connectors/test/Connector.EFCore.Test/MySqlDbContextOptionsExtensionsTest.cs

```diff
@@ -2,22 +2,20 @@
 // The .NET Foundation licenses this file to you under the Apache 2.0 License.
 // See the LICENSE file in the project root for more information.
 
-using System;
-using System.Collections.Generic;
 using Microsoft.EntityFrameworkCore;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
+using Steeltoe.Connector.EFCore;
+using MySqlDataConnectionAlias = MySql.Data.MySqlClient.MySqlConnection;
```

Comment:
I find it tricky to track what's what, as the namespaces of these external types aren't very helpful. Suggestion: use two aliases to make the distinction clear at their usages:
```c#
using OfficialMySqlConnection = MySql.Data.MySqlClient.MySqlConnection;
using PomeloMySqlConnection = MySqlConnector.MySqlConnection;
```
This would result in a build error when unqualified `MySqlConnection` is used, because there's no namespace import for it (which is a good thing).